### PR TITLE
Fixes after code review

### PR DESCRIPTION
**Pull request by xahgmah**

### CODE REVIEW

File: video_xblock/static/js/studio-edit.js [hunk 1]
```diff
@@ -372,47 +373,37 @@ function StudioEditableXBlock(runtime, element) {
 
         $.ajax(options)
         .done(function(response) {
-            var error_message = response['error_message'];
             var success_message = response['success_message'];
+            var error_message = response['error_message'];
             if (success_message) {
                 if (response.transcripts) {
```
Comment (z4y4ts) [inline video_xblock/static/js/studio-edit.js]:
```
if (success_message && response.transcripts) {
// Do transcripts stuff
```

File: video_xblock/static/js/studio-edit.js [hunk 2]
```diff
@@ -372,47 +373,37 @@ function StudioEditableXBlock(runtime, element) {
...
                         }
                     });
                 }
-                message = success_message;
-                status = SUCCESS;
-            }
-            else if (error_message) {
-                message = error_message;
-                status = ERROR;
+                return showStatus($status_block, SUCCESS, success_message);
```
Comment (z4y4ts) [inline video_xblock/static/js/studio-edit.js]:
```
} else if (success_message) {
// Set message and status
```

File: video_xblock/static/js/studio-edit.js [hunk 3]
```diff
@@ -372,47 +373,37 @@ function StudioEditableXBlock(runtime, element) {
...
+            } else {
```
Comment (z4y4ts) [inline video_xblock/static/js/studio-edit.js]:
```
} else if (error_message) {
// Set message and status
```

File: video_xblock/static/js/studio-edit.js [hunk 4]
```diff
@@ -372,41 +372,37 @@ function StudioEditableXBlock(runtime, element) {
...
+                    // Add a transcript from the 3playmedia only for non exists language
+                    if (!includeLang) {
+                        createTranscriptBlock(item.lang, item.label, transcriptsValue, item.url);
+                        pushTranscript(item.lang, item.label, item.url, '', transcriptsValue);
+                        pushTranscriptsValue(transcriptsValue);
+                    }
+                });
+            }
+
+            if (success_message){
```
Comment (z4y4ts) [inline video_xblock/static/js/studio-edit.js]: Add a space before `{`

File: video_xblock/static/html/studio-edit-tab.html [hunk 5]
```diff
@@ -40,7 +40,7 @@
                 {% elif field.type == 'token' %}
                     {% include 'studio-edit-token.html' %}
                 {% elif field.type == 'threeplaymedia_apikey' %}
-                    {% include 'studio-edit-playmedia_apikey.html' %}
+                    {% include 'studio-edit-playmedia-apikey.html' %}
```
Comment (z4y4ts) [inline video_xblock/static/html/studio-edit-tab.html]: Please name it `3playmedia`, not just `playmedia`

**Comment by z4y4ts** [comment]
Also please start PR title with the capital letter

File: video_xblock/static/html/studio-edit-tab.html (hunk 5 above)
Comment (xahgmah) [inline video_xblock/static/html/studio-edit-tab.html]: Done

File: video_xblock/static/js/studio-edit.js (hunk 4 above)
Comment (xahgmah) [inline video_xblock/static/js/studio-edit.js]: Done

**Pull request merged**